File: src/nomanssave/bk.java
package nomanssave;

import java.awt.event.ActionEvent;
import java.awt.event.ActionListener;

class bk implements ActionListener {
   // $FF: synthetic field
   final bd dP;
   // $FF: synthetic field
   private final Application bv;

   bk(bd var1, Application var2) {
      this.dP = var1;
      this.bv = var2;
   }

   public void actionPerformed(ActionEvent var1) {
      if (bd.a(this.dP) != null) {
         this.bv.b(bd.a(this.dP));
      }
   }
}

File: NMSSaveEditor/nomanssave/lower/bk.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Windows.Forms;

namespace NMSSaveEditor
{

public class bk : ActionListener {
   bd dP;
   public Application bv;

   public bk(bd var1, Application var2) {
      this.dP = var1;
      this.bv = var2;
   }

   public void actionPerformed(EventArgs var1) {
      if (bd.a(this.dP) != null) {
         this.bv.b(bd.a(this.dP));
      }
   }
}

}
